include/cxxopts.hpp

```diff
@@ -755,19 +778,23 @@ inline bool IsFalseText(const std::string &text)
   return !result.empty();
 }
 
-inline std::pair<std::string, std::string> SplitSwitchDef(const std::string &text)
+// Gets the option names specified via a single, comma-separated string,
+// and returns the separate, space-discarded, non-empty names
+// (without considering which or how many are single-character)
+inline OptionNames split_option_names(const std::string &text)
 {
-  std::match_results<const char*> result;
-  std::regex_match(text.c_str(), result, option_specifier);
-  if (result.empty())
+  if (not std::regex_match(text.c_str(), option_specifier))
```

Comment:
can you use `!` instead of `not`

Reply:
I could... although personally I've long switched to only using `and`, `or` and `not` to make my code more natural-language like.